specification/storage/Microsoft.BlobStorage/routes.tsp

```diff
@@ -138,7 +138,7 @@ namespace Storage.Blob {
     /** The media type of the body of the response. For batch requests, this is multipart/mixed; boundary=batchresponse_GUID */
     #suppress "@typespec/http/content-type-ignored" "Template for existing API"
     @header("Content-Type")
-    contentType: "multipart/mixed";
+    contentType: "application/octet-stream";
```

Comment:
@jhendrixMSFT didnt the original swagger specify multipart/mixed. Was that wrong?

Reply:
No. This is a temporary workaround until we have support for `multipart/mixed` in the Rust emitter (https://github.com/Azure/typespec-rust/issues/349).

Reply:
If there's a way to constrain this to just Rust, I can make that change.

Reply:
Maybe a better fix it to reinstate `multipart/mixed` and skip the offending operations for Rust. I'm looking into this now.

Reply:
I have this working, will push the change shortly.

Reply:
Done, see https://github.com/Azure/azure-rest-api-specs/commit/201a318efa3da6cc4dcf3e22c8e12ad3c43400eb